src/libraries/System.Drawing.Common/src/System/Drawing/Image.Windows.cs

```diff
@@ -229,7 +229,7 @@ public void Save(Stream stream, ImageCodecInfo encoder, EncoderParameters encode
 
                 if (_rawData != null)
                 {
-                    ImageCodecInfo rawEncoder = RawFormat.FindEncoder();
+                    ImageCodecInfo? rawEncoder = RawFormat.FindEncoder();
                     if (rawEncoder != null && rawEncoder.Clsid == g)
                     {
                         stream.Write(_rawData, 0, _rawData.Length);
```

Comment:
Below row 261, 285 parameter `EncoderParameters encoderParams` is nullable for `SaveAdd(...)` overloads